addons/bestja_volunteer/models.py

```diff
@@ -126,48 +135,63 @@ def sync_group(self, group, domain):
         })
 
     def check_password_rules(self, password):
```

Comment:
I was actually playing with the name change. I like to start boolean method names with "is". But I feel any name with "password is valid" may be misleading - sounds like method for checking validity of a password, i.e. authentication. So I ended up leaving it unchanged. Any other name ideas?